tempesta_fw/connection.h

```diff
@@ -197,15 +346,25 @@ tfw_connection_put(TfwConnection *conn)
 	if (conn->destructor)
 		conn->destructor(conn);
 }
+static inline void
+tfw_cli_conn_put(TfwCliConn *cli_conn)
+{
+	tfw_connection_put((TfwConn *)cli_conn);
+}
+static inline void
+tfw_srv_conn_put(TfwSrvConn *srv_conn)
+{
+	tfw_connection_put((TfwConn *)srv_conn);
+}
```

Comment:
The use of inline functions is intentional as these provide type checking. I consider this feature important as these conversions from one type of connection to another type may yield a bug that would be very difficult to discover. Macros do not provide that feature, otherwise I would agree with you. Would you still like me to replace these with macros?

Also, the missing empty lines mentioned in the comments above are also intentional. These one-line functions are visibly and visually grouped together as they are basically the same. Would you suggest a different solution to this type of grouping? Or would you just like me break this type of grouping and separate all functions with an empty line?